indico/htdocs/js/palette.js

```diff
@@ -27,4 +27,9 @@
 
     // colors for specific purposes
     Palette.highlight = Palette.blue;
+
+    // colors for the bubble avatar in abstract page
```

Comment:
`// colors for user-specific avatar bubbles`

We might reuse this in other places so I'd rather not mention abstracts